internal/pkg/template/templates/environment/cf.yml

```
@@ -416,7 +416,8 @@ Resources:
 {{- end}}
 {{include "cfn-execution-role" . | indent 2}}
 {{include "environment-manager-role" . | indent 2}}
-{{- if not .ImportCertARNs}}
+{{- if not .PublicImportedCertARNs}}
+{{- if not .PrivateImportedCertARNs}}
```

I think we want to create to create the resource regardless if the customer has imported private certs since they get used for the public lb
```suggestion
{{- if not .PublicImportedCertARNs}}
```